src/classes/RD2_EnablementService_TEST.cls

```diff
@@ -132,7 +132,9 @@ public with sharing class RD2_EnablementService_TEST {
         npe03__Recurring_Donation__c rd = new TEST_RecurringDonationBuilder()
             .withContact(contact.Id)
             .withAmount(100)
+            .withInstallmentPeriodYearly()
```

Comment:
@RadSFDeveloper , I changed to use installmentperiodyearly so that day of month might not be required to add in the test. Default is monthly and do you want to change it like that?

Reply:
Using yearly instead of monthly does not bring any difference except to ensure test RDs comply to new validation rules. Is there any benefit/purpose of using yearly instead of default monthly?

Reply:
Just setting to be not required, not sure about benefit.

Reply:
I will change it.